styles.css

```diff
@@ -464,14 +466,50 @@ p {
   }
 
   .container {
+    max-width: 1440px;
     margin: 0 auto;
-    padding: 20px;
+  }
+
+  /* navbar */
+  .navbar ul {
+    display: flex;
+  }
+
+  .navbar ul a {
+    font-weight: 500;
+    font-size: 15px;
+    color: #344563;
+    margin-right: 50px;
+  }
+
+  .navbar i {
+    display: none;
+  }
+
+  /* banner */
+  .banner {
+    position: static;
+    background-image: url("images/bg/header-bg.png");
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: center;
+    top: 70px;
+    background-color: #fff;
+    border-bottom-left-radius: 160px;
+    height: 846px;
+    width: 100%;
+  }
+
+  .banner .banner-text {
+    /* padding: 150px 11rem; */
+    padding: 5rem 9rem;
   }
 
   /* Work */
   .work .card {
     flex-direction: row;
-    gap: 40px;
+    gap: 25px;
+    height: 351px;
   }
 
```

Comment:
- [ ] You did a good job @evansnyamekye 🥇, however, as I can see you gave your cards a `box-shadow` 
https://github.com/evansnyamekye/Post-index/blob/f7a77daf01eb8622da8a5a069050623872b666bd/styles.css#L185

But, I'll suggest you remove that line to stick to the given design. However the `box-shadow` will be used when we `hover` the card when are in the desktop version, it supposes to be something like this: 

````css
@media only screen and (min-width: 768px) {
       .
       .
       .

       .work .card {
            flex-direction: row;
            cursor: pointer;
            gap: 25px;
            height: 351px;
       }

      .work .card:hover {
            box-shadow: 0 10px 25px 0 rgba(0, 0, 0, 0.103);
            border: #A7AEFF solid 1px;
      }

      .
      .
      .
}

````

Kindly, fix it 😃